Runtime/Interaction/SnapZone.cs

```diff
@@ -91,7 +94,27 @@ public Material ValidationMaterial
 
                 return validationMaterial;
             }
-            set { validationMaterial = value; }
+            set => validationMaterial = value;
+        }
+        
+        [SerializeField]
+        private Material invalidMaterial;
```

Comment:
Can this be accessed from the editor? Whichever the answer is, is it intended?

Reply:
yes, you should be able to exchange the invalid material

Reply:
Tooltip?

Reply:
tooltipped